docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
+
+services:
+  web:
+    build: .
+    command: uvicorn app.main:app --reload --workers 1 --host 0.0.0.0 --port 8000
+    volumes:
+      - ./app/:/usr/src/app/app/
+    ports:
+      - ${PORT}:8000
+    environment:
```

Comment:
Maybe we should said in the doc that potential contributors should created their own `.env` file ? Because it might not be obvious, in that case we should also add `dotenv` in the project requirements ?

Reply:
Just added it to the README & set the `dotenv`! But about the  `.env` actually, you can also only have set those as env variables!

Reply:
Perfect ! all good for me :)